src/main/java/teammates/ui/pagedata/StudentFeedbackResultsPageData.java

```diff
@@ -235,4 +238,15 @@ private FeedbackResultsResponseTable createResponseTable(FeedbackQuestionAttribu
         }
         return responsesForRecipient;
     }
+
+    /**
+     * Remove the anonymous hash from anonymous participant name or email, if any.
```

Comment:
Removes

Reply:
Done

Reply:
He meant change `Remove` to `Removes`

Reply:
I guess removing the javadoc altogether is fine too since this is a temporary method.

Reply:
@bqnguyen94 I'll leave it up to you whether you want to add the javadoc back in (but if you do make sure the tense is `Removes` not `Remove`). You can merge your own PR :)